onebot/plugins/lastfm.py

```diff
@@ -306,13 +314,28 @@ def fetch_extra_trackinfo(self, username, info):
         if 'userplaycount' in api_result:
             info['playcount'] = int(api_result['userplaycount'])
 
-        if 'toptags' in api_result and 'tag' in api_result['toptags']:
-            taglist = api_result['toptags']['tag']
-            if not isinstance(api_result['toptags']['tag'],
-                              list):  # pragma: no cover
-                self.log.warning("Tags is not a list: %r", taglist)
-            else:
-                info['tags'] = [tag['name'] for tag in taglist]
+        # if 'toptags' in api_result and 'tag' in api_result['toptags']:
+        #     taglist = api_result['toptags']['tag']
+        #     if not isinstance(api_result['toptags']['tag'],
+        #                       list):  # pragma: no cover
+        #         self.log.warning("Tags is not a list: %r", taglist)
+        #     else:
+        #         info['tags'] = [tag['name'] for tag in taglist]
```

Comment:
Why comment this out. Either delete it or (alternatively) turn it into a configurable thing: either use mbrainz tags or use last.fm tags.